appveyor.yml

```diff
@@ -129,4 +129,24 @@ on_success:
   - set ARTIFACT=%GROONGA_INSTALL_FOLDER%.zip
   - 7z a %ARTIFACT% %FULL_GROONGA_INSTALL_FOLDER%
   - ps: Push-AppveyorArtifact $Env:ARTIFACT
+  - if "%ARCH%" == "amd64" set ARCH=x64
+  - ps: |
+      if ($Env:VS_VERSION -eq "15") {
+        $Env:REDIST_VERSION = (Get-Content "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Auxiliary\Build\Microsoft.VCRedistVersion.default.txt")
+        $Env:VCREDIST_VCRUNTIME = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\$Env:REDIST_VERSION\$Env:ARCH\Microsoft.VC141.CRT\vcruntime140.dll"
+        $Env:VCREDIST_MSVCP = "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\$Env:REDIST_VERSION\$Env:ARCH\Microsoft.VC141.CRT\msvcp140.dll"
```

Comment:
Is `vcruntime140.dll` needless?

Reply:
vcrruntime140.dll is CRT runtime, so it is required.